gen_cert.py

```diff
@@ -251,8 +251,7 @@ def interstitial_factory():
         # id. If TEMPLATEFILE is given in cert-data, use that template instead.
         # If template_pdf comes in as part of object initialization (e.g., from
         # the xqueue), then use that instead.
-        if not template_pdf:
-            template_pdf = cert_data.get('TEMPLATEFILE', None)
+        template_pdf = cert_data.get('TEMPLATEFILE', template_pdf)
```

Comment:
So, Im' not quite clear what problem this fixes, and this change invalidates the documentation lines above.